docs/api/baseplate/clients/cassandra.rst

```diff
@@ -0,0 +1,62 @@
+``baseplate.clients.cassandra``
+===============================
+
+.. automodule:: baseplate.clients.cassandra
+
+This integration supports both the base Python Cassandra driver and the
+Cassandra ORM, CQLMapper.
+
+To integrate it with your application, add the appropriate client declaration
+to your context configuration::
+
+   baseplate.configure_context(
+      app_config,
+      {
+         ...
+         "foo": CassandraClient(),
+         ...
+      }
+   )
+
+configure it in your application's configuration file:
+
+.. code-block:: ini
+
+   [app:main]
+
+   ...
+
+   # required: a comma-delimited list of hosts to contact to find the ring
+   foo.contact_points = cassandra-01.local, cassandra-02.local
+
+   # optional: the port to connect to on each cassandra server
```

Comment:
🔕 worth specifying the default if no value is provided? Or is that best done/already done elsewhere? I always have to dig up these Cassandra ports after longer periods of not having touched Cass or an app that uses Cass.

Reply:
Updated. It's 9042. I can't keep track of 'em either.